lib/config_cat.ex

```diff
@@ -0,0 +1,187 @@
+defmodule ConfigCat do
+  use GenServer
+
+  alias ConfigCat.{FetchPolicy, Rollout}
+  alias HTTPoison.Response
+
+  require Logger
+
+  @base_url "https://cdn.configcat.com"
+  @base_path "configuration-files"
+  @config_filename "config_v4.json"
```

Comment:
We are using now the new config_v5.json format. There are some breaking changes in that (structure modified).

Reply:
Extracted to #18